apps/ui/src/components/App/Sidebar.vue

```diff
@@ -10,7 +10,7 @@ const followedSpacesStore = useFollowedSpacesStore();
     class="w-[72px] flex flex-col border-r fixed left-0 inset-y-0 text-center h-screen"
   >
     <router-link :to="{ name: 'my-home' }" class="h-[72px] block">
-      <IH-stop class="inline-block my-4 size-[32px] text-skin-link" />
+      <IC-zap class="inline-block my-4 size-[36px] text-skin-link" />
```

Comment:
Like this, a bit bigger and centered with the topnav
```suggestion
      <IC-zap class="inline-block my-[18px] [ size-[40px] text-skin-link" />
```